pkg/protocol/metricstracker/trivialmetricstracker/trivialmetricstracker.go

```diff
@@ -0,0 +1,124 @@
+package trivialmetricstracker
+
+import (
+	"sync"
+	"time"
+
+	"github.com/iotaledger/hive.go/runtime/event"
+	"github.com/iotaledger/hive.go/runtime/module"
+	"github.com/iotaledger/iota-core/pkg/metrics"
+	"github.com/iotaledger/iota-core/pkg/protocol/engine"
+	"github.com/iotaledger/iota-core/pkg/protocol/engine/blocks"
+	"github.com/iotaledger/iota-core/pkg/protocol/engine/notarization"
+	"github.com/iotaledger/iota-core/pkg/protocol/metricstracker"
+)
+
+type (
+	isBootstrappedFunc func() bool
+)
+
+type MetricsTracker struct {
+	metrics *metrics.ServerMetrics
+
+	oldBlockCount          uint64
+	oldConfirmedBlockCount uint64
+	oldTime                int64
+
+	bps               float64
+	bpsLock           sync.RWMutex
+	cbps              float64
+	cbpsLock          sync.RWMutex
+	confirmedRate     float64
+	confirmedRateLock sync.RWMutex
+
+	isBootstrappedFunc isBootstrappedFunc
+
+	module.Module
+}
+
+// NewProvider creates a new MetricsTracker provider.
+func NewProvider() module.Provider[*engine.Engine, metricstracker.MetricsTracker] {
```

Comment:
Why is this an engine module instead of an app component?

Reply:
You're right, this can be an app component.